index.js

```diff
@@ -272,21 +272,24 @@ import {
   removePackage as _removePackage,
   reloadPackage as _reloadPackage,
   applyConfig as _applyPackageConfig,
-  getPackages as _getPackages
+  getPackages as _getPackages,
+  searchPackage as _searchPackage
 } from './src/packages.js'
 function importPackage(packageURL) { return _importPackage(defaultSystem, packageURL); }
 function registerPackage(packageURL) { return _registerPackage(defaultSystem, packageURL); }
 function removePackage(packageURL) { return _removePackage(defaultSystem, packageURL); }
 function reloadPackage(packageURL) { return _reloadPackage(defaultSystem, packageURL); }
 function getPackages(moduleNames) { return _getPackages(defaultSystem); }
 function applyPackageConfig(packageConfig, packageURL) { return _applyPackageConfig(defaultSystem, packageConfig, packageURL); }
+function searchPackage(packageURL, searchString) { return _searchPackage(defaultSystem, packageURL, searchString); }
```

Comment:
I think it would be better to have a `Package` class which offers methods such as `search`.